styles.css

```diff
@@ -0,0 +1,125 @@
+
+*{
+    margin:0;
+    padding:.5rem;
+    box-sizing: border-box;
+
+}
+body{
+    font-family: 'Roboto', sans-serif;
+    font-size: 1.2rem;
+    font-weight: 400;
+    line-height: 1.2;
+    color: #212529;
+    text-align: left;
+    width:100vw;
+    height:100vh;
+    background-color:#f8f9fa;
+}
+
+.hero{
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+    text-align: center;
+}
+
+h1, h2{
+    font-weight: 400;
+    font-size: 2.5rem;
+    line-height: 1.2;
+}
+h3{
+    font-size: 1.2rem;
+    font-weight: 800;
+}
+.card p{

```

Comment:
Empty selector

Reply:
Abul ... Thanks for the review ... most grateful. I will go back to the drawing board and resolve. Cheers Dave